### PR TITLE
feat: Change Bandwidth

### PR DESCRIPTION
**Pull request by vishwajeetraj11**

### CODE REVIEW

File: src/lib/page-components/oyster/inventory/modals/ChangeBandwidthModal.svelte [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+	import InputCard from '$lib/atoms/cards/InputCard.svelte';
+	import Text from '$lib/atoms/texts/Text.svelte';
+
+	export let modalFor: string;
+	export let jobData: OysterInventoryDataModel;
+
+	let checkboxAcknowledge = false;
+	let currentBandwidthUnit = DEFAULT_BANDWIDTH_UNIT;
+	let newBandwidthUnit = DEFAULT_BANDWIDTH_UNIT;
+	$: submitLoading = false || rateStatus === 'pending';
```
Comment (souvikmishra) [inline src/lib/page-components/oyster/inventory/modals/ChangeBandwidthModal.svelte:47]: maybe move this to the end of file to make it more readable and predictable

File: src/lib/controllers/contract/oysterCredit.ts [hunk 2]
```diff
@@ -180,6 +212,39 @@ export async function finaliseRateReviseOysterCreditJob(jobId: BytesLike) {
...
+		};
+
+		const { txn } = await createTransaction(
+			() => oysterCreditContract.jobReviseRateFinalize(jobId),
+			initiateTxnMessage,
+			successTxnMessage,
+			errorTxnMessage,
+			parentFunctionName,
+			titles,
+			{ disableToastDescription: true }
```
Comment (souvikmishra) [inline src/lib/controllers/contract/oysterCredit.ts:239]: should this be handled inside the toast component itself? seems a bit odd to handle it from here...what do you think?

File: src/lib/utils/helpers/oysterHelpers.ts [hunk 3]
```diff
@@ -579,3 +581,12 @@ export const transformOysterJobDataToInventoryDataModel = (
 	};
 	return newJob;
 };
+
+export const calculatBandwidthRate = (bandwidth: string, region: string, bandwidthUnit: string) => {
+	return (
```
Comment (souvikmishra) [inline src/lib/utils/helpers/oysterHelpers.ts:586]: could be made more readable, upto you

File: src/lib/controllers/contract/oysterCredit.ts (hunk 2 above)
Comment (vishwajeetraj11) [reply src/lib/controllers/contract/oysterCredit.ts:239]: this is handled down to toast component, as long as passed description is empty or not passed at all it won't render description.

createTransaction is used widely and in most cases description is required.

**Review by souvikmishra — APPROVED** [review]

**Pull request merged**